Bottom: c597c50a3b5b3f6bc4504eac7ae05a021a0ef6d2
Top:    ebd72ed5386bc3f20c3ec7fe8769c59c498471da
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 02:29:59 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
